internal/scorecard/alpha/bundle.go

```diff
@@ -18,20 +18,21 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 
 	"k8s.io/apimachinery/pkg/util/rand"
 )
 
 // getBundleData tars up the contents of a bundle from a path, and returns that tar file in []byte
-func (o Scorecard) getBundleData() (bundleData []byte, err error) {
+func (o Scorecard) GetBundleData() (bundleData []byte, err error) {
```

Comment:
Since this is only used to create the ConfigMap for the PodTestRunner, I would recommend that it remain unexported and be relocated as a method on the `PodTestRunner` used as part of the `PodTestRunner.Initialize()` function if that's the route we go.

Reply:
its now unexported on the PodTestRunner and called in the Initialize()